website/docs/plugins/ray_launcher.md

```diff
@@ -88,7 +88,7 @@ $ tree -L 1
 ├── model
 └── train.py
 
-$ python train.py hydra/launcher=ray_aws mode=aws random_seed=1,2,3 -m
+$ python train.py hydra/launcher=ray_aws +mode=aws random_seed=1,2,3 -m
```

Comment:
Using + here is not the best example.
Also - mode is a bit too generic.
maybe ray_mode?

You can also add it to your defaults list:
```yaml
defaults:
  - hydra/launcher: ray
  - ray_mode: local
```

Reply:
sounds good. updated.